internals/timeutil/schedule.go

```diff
@@ -395,9 +394,7 @@ func randDur(a, b time.Time) time.Duration {
 	return randutil.RandomDuration(dur)
 }
 
-var (
-	timeNow = time.Now
-)
+var timeNow = time.Now
```

Comment:
Looks like this only so that it can be mocked out in tests.

Is this a Go idiom, or does this dependency injection point deserve a comment?